docs/tutorials/cpp.md

```diff
@@ -73,11 +76,9 @@ int main() {
 }
 ```
 
-## no_std and embedded platforms
+## Embedded platforms (`no_std`)
 
-Users wishing to use ICU4X on a `no_std` platform will need to write their own crate depending on `icu_capi` that fills in an allocator and panic hooks, similar to what we do in our [freertos port]. ICU4X is happy to accept such ports upstream.
-
-The `icu_capi_staticlib` crate additionally supports an `x86tiny` for highly size-optimized builds, to be run with `cargo panic-abort-build` (shorthand for `build` with `-Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort`). This requires a nighly compiler; it has been tested to work on `nightly-2022-04-05`.
+Building `icu_capi_staticlib` for embedded systems (i.e. without `libc`) requires some additional parameters. The feature `x86tiny` includes an allocator and removes Rust's panicking code. This should be built with `-Z build-std=std,panic_abort -Z build-std-features=panic_immediate_abort`, which requires a nightly compiler (`cargo +nightly ...`).
```

Comment:
I still think the sentence about an allocator and panic hooks is important. x86tiny is mostly for testing.

Please can we just not change this section?

Reply:
+1; x86tiny exists because of the size test, but it isn't necessarily designed for external use; better is for people to set up their own handlers.